docs/description.md

```diff
@@ -377,6 +376,88 @@ computations.
 
 🚧 Under construction 🚧
 
+### Use AWS Bedrock Models
+
+Another alternative if you don't have the necessary GPU to run local models, is using AWS Bedrock models. The default chosen models in Bedrock are Anthropic Claude v2 (LLM) and Titan Embeddings G1 - Text (embedding). You can navigate to `settings-bedrock.yaml` and change the `llm_modelid` and `embedding_modelid` parameters. You can find the available models and their IDs in the link https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html
+
+In order to use the Bedrock models, you need to do the following configurations.
+
+#### Install AWS CLI
```

Comment:
Given there is already Sagemaker support, you could document this as common documentation in fern/docs/pages/manual/llms.mdx